lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2007/179d_ashrae_90_1_2007/179d_ashrae_90_1_2007.AirLoopHVAC.rb

```diff
@@ -0,0 +1,10 @@
+class ACM179dASHRAE9012007
+  # @!group AirLoopHVAC
+
+  # Set default fan curve to be VSD with static pressure reset
```

Comment:
```suggestion
  # Set default fan curve to be VSD with static pressure reset
  # NOTE: 179D overrides it because we want the use the proper fan coefs, 
  # and not the ones from 'Multi Zone VAV with VSD and SP Setpoint Reset'
```